doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -44,12 +78,33 @@ Provisioning the device
 A device model must be registered with Google to work as a Fast Pair Provider.
 The data is used for procedures defined by the Fast Pair standard.
 
+.. _ug_bt_fast_pair_provisioning_register:
+
 Registering Fast Pair Provider
 ==============================
 
 See the official `Fast Pair Model Registration`_ documentation for information on how to register the device and obtain the Model ID and Anti-Spoofing Public/Private Key pair.
 Alternatively, you can use the debug Model ID and Anti-Spoofing Public/Private Key pair obtained by Nordic Semiconductor for the development purposes.
-See the :ref:`fast_pair_input_device` sample and the :ref:`nrf_desktop` for details about the debug Fast Pair Providers registered by Nordic.
+See the following samples and applications for details about the debug Fast Pair Providers registered by Nordic:
+
+* the :ref:`fast_pair_input_device` sample
+* the :ref:`nrf_desktop` application
+
+.. _ug_bt_fast_pair_provisioning_register_device_type:
+
+Device type
+-----------
+
+When registering the device in the Google Nearby Devices console, go to the **Fast Pair** protocol configuration panel, and in the **Device Type** field select an option that matches your application's use case.
+The chosen device type also determines the optional feature set that you can support in your use case.
+You declare support for each feature by selecting the **true** option.
+
+.. note::
+   Ensure you make an informed decision when selecting the device type, as it has a significant impact on the Fast Pair Seeker behavior in relation to your Provider's device.
+
+The Fast Pair standard implementation in the |NCS| actively supports the following device types and use cases:
+
+* Input device (see the :ref:`fast_pair_input_device` sample)
```

Comment:
```suggestion
The Fast Pair standard implementation in the |NCS| actively supports the Input device type (see the :ref:`fast_pair_input_device` sample) and its use cases.
```
Since it is just a single entry. You can leave it as is if you plan to add more to this list.

Reply:
Discussed offline. Let's leave as it is since we expect more samples and device types to be supported soon